slices/tomcat10.yaml

```
@@ -0,0 +1,69 @@
+package: tomcat10
+
+essential:
+  - tomcat10_copyright
+
+slices:
+  core:
+    essential:
+      - coreutils_bins
```

Assuming this works the same in `25.05` and `25.10`:

```suggestion
      - coreutils_delaying
      - coreutils_directory-listing
      - coreutils_dirname
      - coreutils_echo
      - coreutils_expr
      - coreutils_output-of-entire-files
      - coreutils_rm-utility
      - coreutils_special-file-types
      - coreutils_system-context
      - coreutils_touch
      - coreutils_working-context
```

Please double check